bip-musig2.mediawiki

```diff
@@ -713,8 +713,7 @@ An exception to this rule is <code>MAJOR</code> version zero (0.y.z) which is fo
 The <code>MINOR</code> version is incremented whenever the inputs or the output of an algorithm changes in a backward-compatible way or new backward-compatible functionality is added.
 The <code>PATCH</code> version is incremented for other changes that are noteworthy (bug fixes, test vectors, important clarifications, etc.).
 
-* '''1.0.0''' (2022-10-03): Submit draft BIP to the BIPs repository
```

Comment:
Strictly speaking this would be require MAJOR version change, since the inputs and outputs are incompatible. We could do view it as MINOR change since this only affects data that remains local to the signer. But OTOH it's also backwards incompatible because it prohibits some applications (in particular applications where you don't know the pubkey at nonce gen time).

One alternative would be that we retroactively rename 1.0.0 to 1.0.0-rc1 and add rc2 with this PR.

Reply:
> One alternative would be that we retroactively rename 1.0.0 to 1.0.0-rc1 and add rc2 with this PR.

done